electron/js/menu/developer.js

```diff
@@ -42,32 +43,72 @@ var devToolsTemplate = {
 
 var devProductionTemplate = {
   label: 'Production',
-  click: function() {getPrimaryWindow().loadURL(config.PRODUCTION_URL);},
+  type: 'radio',
+  checked: env === config.PROD,
+  click: function() {
+    getPrimaryWindow().loadURL(config.PROD_URL);
+    init.save('env', config.PROD);
+  },
+};
+
+var devInternalTemplate = {
+  label: 'Internal',
+  type: 'radio',
+  checked: env === config.INTERNAL,
+  click: function() {
+    getPrimaryWindow().loadURL(config.INTERNAL_URL);
+    init.save('env', config.INTERNAL);
+  },
 };
 
 var devStagingTemplate = {
   label: 'Staging',
-  click: function() {getPrimaryWindow().loadURL(config.STAGING_URL);},
+  type: 'radio',
+  checked: env === config.STAGING,
+  click: function() {
+    getPrimaryWindow().loadURL(config.STAGING_URL);
```

Comment:
we could simplify the code by just having one handler that changes the env. right now we have 7x the same code